### PR TITLE
Update Set-CsCallingLineIdentity.md

### PR DESCRIPTION
**Pull request by JensTrier**
Added the 2 new parameters ResourceAccount and CompanyName

### CODE REVIEW

**Review by get-itips — COMMENTED** [review]
Checked in 2.2.0-preview and the parameters are still not there, is this going live in the next preview version?

**Comment by JensTrier** [comment]
> Checked in 2.2.0-preview and the parameters are still not there, is this going live in the next preview version?

Yes, they are not in Teams PS Module and will be added in the next preview version. Hopefully by end of Month. They are in SfBO PS.

Should I add a note about this?

File: skype/skype-ps/skype/Set-CsCallingLineIdentity.md [hunk 1]
```diff
@@ -206,6 +231,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ResourceAccount
+This parameter specifies the ObjectId of a resource account/online application instance used for Teams Auto Attendant or Call Queue. The outgoing PSTN call will use the phone number defined on the resource account as caller id. For more information about resource accounts please see https://docs.microsoft.com/en-us/microsoftteams/manage-resource-accounts.
```
Comment (dariomws) [inline skype/skype-ps/skype/Set-CsCallingLineIdentity.md:235]:
```suggestion
This parameter specifies the ObjectId of a resource account/online application instance used for Teams Auto Attendant or Call Queue. The outgoing PSTN call will use the phone number defined on the resource account as caller id. For more information about resource accounts please see https://docs.microsoft.com/microsoftteams/manage-resource-accounts.
```

**Review by dariomws — APPROVED** [review]
Approve after the link is fixed.

**Review by tseward — APPROVED** [review]

**Pull request merged**